spring-boot-devtools/src/main/java/org/springframework/boot/devtools/env/DevToolsPropertyDefaultsPostProcessor.java

```diff
@@ -42,31 +42,31 @@
 @Order(Ordered.LOWEST_PRECEDENCE)
 public class DevToolsPropertyDefaultsPostProcessor implements EnvironmentPostProcessor {
 
-	private static final Map<String, Object> PROPERTIES;
+	private static final Map<String, Object> properties;
```

Comment:
This could become a `Map<String, String>` and remain as `PROPERTIES`.

Reply:
@wilkinsona If the type is changed, it's not matched with the type being used: `MapPropertySource(String name, Map<String, Object> source)`

I guess you don't want to copy for it.

Reply:
Sorry, you're right. It'll need to be changed to `properties` as you've proposed.